src/Term_Command.php

```diff
@@ -636,6 +636,95 @@ public function recount( $args ) {
 		}
 	}
 
+	/**
+	 * Migrate a term of a taxonomy to another taxonomy.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <term>
+	 * : Slug or ID of the term to migrate.
+	 *
+	 * [--by=<field>]
+	 * : Explicitly handle the term value as a slug or id.
+	 * ---
+	 * default: id
+	 * options:
+	 *   - slug
+	 *   - id
+	 * ---
+	 *
+	 * [--from=<taxonomy>]
```

Comment:
This should not be optional, and we should require users to always explicitly specify the `--from` and `--to` taxonomies.